c11threads.h

```diff
@@ -60,13 +118,286 @@ enum {
+{
+	_mtx_destroy_win32(mtx);
+}
+
+int _mtx_lock_win32(mtx_t *mtx);
+static inline int mtx_lock(mtx_t *mtx)
+{
+	return _mtx_lock_win32(mtx);
+}
+
+int _mtx_trylock_win32(mtx_t *mtx);
+static inline int mtx_trylock(mtx_t *mtx)
+{
+	return _mtx_trylock_win32(mtx);
+}
+
+int _mtx_timedlock_win32(mtx_t *mtx, const struct timespec *ts);
+static inline int mtx_timedlock(mtx_t *mtx, const struct timespec *ts)
+{
+	return _mtx_timedlock_win32(mtx, ts);
+}
+
+int _mtx_unlock_win32(mtx_t *mtx);
+static inline int mtx_unlock(mtx_t *mtx)
+{
+	return _mtx_unlock_win32(mtx);
+}
+
+/* ---- condition variables ---- */
+
+#ifndef C11THREADS_SUPPORT_WINNT_OLDER_THAN_VISTA
+int _cnd_init_win32(cnd_t *cond);
+static inline int cnd_init(cnd_t *cond)
+{
+	return _cnd_init_win32(cond);
+}
+
+static inline void cnd_destroy(cnd_t *cond)
+{
+	(void)cond;
+}
+
+int _cnd_signal_win32(cnd_t *cond);
+static inline int cnd_signal(cnd_t *cond)
+{
+	return _cnd_signal_win32(cond);
+}
+
+int _cnd_broadcast_win32(cnd_t *cond);
+static inline int cnd_broadcast(cnd_t *cond)
+{
+	return _cnd_broadcast_win32(cond);
+}
+
+int _cnd_wait_win32(cnd_t *cond, mtx_t *mtx);
+static inline int cnd_wait(cnd_t *cond, mtx_t *mtx)
+{
+	return _cnd_wait_win32(cond, mtx);
+}
+
+int _cnd_timedwait_win32(cnd_t *cond, mtx_t *mtx, const struct timespec *ts);
+static inline int cnd_timedwait(cnd_t *cond, mtx_t *mtx, const struct timespec *ts)
+{
+	return _cnd_timedwait_win32(cond, mtx, ts);
+}
+#endif
+
+/* ---- thread-specific data ---- */
+
+int _tss_create_win32(tss_t *key, tss_dtor_t dtor);
+static inline int tss_create(tss_t *key, tss_dtor_t dtor)
+{
+	return _tss_create_win32(key, dtor);
+}
+
+void _tss_delete_win32(tss_t key);
+static inline void tss_delete(tss_t key)
+{
+	_tss_delete_win32(key);
+}
+
+int _tss_set_win32(tss_t key, void *val);
+static inline int tss_set(tss_t key, void *val)
+{
+	return _tss_set_win32(key, val);
+}
+
+void *_tss_get_win32(tss_t key);
+static inline void *tss_get(tss_t key)
+{
+	return _tss_get_win32(key);
+}
+
+/* ---- misc ---- */
+
+#ifdef C11THREADS_SUPPORT_WINNT_OLDER_THAN_VISTA
+void _call_once_win32_legacy(once_flag *flag, void (*func)(void));
+static inline void call_once(once_flag *flag, void (*func)(void))
+{
+	_call_once_win32_legacy(flag, func);
+}
+#else
+void _call_once_win32(once_flag *flag, void (*func)(void));
+static inline void call_once(once_flag *flag, void (*func)(void))
+{
+	_call_once_win32(flag, func);
+}
+#endif
+
+#ifdef C11THREADS_NO_TIMESPEC_GET
+int _timespec_get_win32(struct timespec *ts, int base);
+static inline int timespec_get(struct timespec *ts, int base)
+{
+	return _timespec_get_win32(ts, base);
+}
+#endif
+
+#else /* !defined(_WIN32) || defined(C11THREADS_PTHREAD_WIN32) */
 
 /* ---- thread management ---- */
 
 static inline int thrd_create(thrd_t *thr, thrd_start_t func, void *arg)
 {
 	int res = pthread_create(thr, 0, (void*(*)(void*))func, arg);
-	if(res == 0) {
+	if (res == 0) {
```

Comment:
Why are there a bunch of whitespace changes from this point on?

Reply:
Visual Studio seems to have reformatted those lines. I'll remove the spaces in another commit.